### ISSUE TITLE
Vulcanizing break action button.

### ISSUE THREAD

**Issue by PiotrekPaprocki** [PredixDev/px-alert-message#22]
Hi, 

Vulcanizing break action button (no action after click, no icon on button). Does anyone noticed this or it's issue related only to my app (i.e. conflicts with other components, styles) or to vulcanizing itself?

Thanks

**Comment by runn-vermel** [comment]
Hi @PiotrekPaprocki 

I suspect it's only in your app - we've had this vulcanized up until very recently (we've moved our hosting to Http2, so we removed the vulcanize step from our production flow). 

have you tried re-producing this outside of your app?

**Comment by PiotrekPaprocki** [comment]
Thx for the hint. I've inspected my problem more deeply and it seems that vulcanizing add a path to the `action` attribute value. The same as created in each dom-module by vulcanization process i.e. `assetpath="component/path"`, so after vulcanize instead of `action="dismiss"` I got `action="component/path/dismiss"`.

For quick solution I just added property 'action' with value "dismiss" and use it in `action="{{action}}"` attribute instead of plain String.
